js/src/modals/Verification/email-store.js

```diff
@@ -23,7 +23,9 @@ import VerificationStore, {
 } from './store';
 import { isServerRunning, postToServer } from '../../3rdparty/email-verification';
 
-const EMAIL_VERIFICATION = 7; // id in the `BadgeReg.sol` contract
+// ids in the `BadgeReg.sol` contract
+const TESTNET_EMAIL_VERIFICATION = 7;
```

Comment:
This really is dangerous. It means that we need to do code upgrades when we deploy a new version and re-register. The code should just be able to get the info from the network, not via hard-coded ids.

In addition, if we have to deploy the BadgeReg on e.g. ETC we would need to come back here.

I don't have a much better idea since I'm not 100% familiar with what is available, but there must be other solutions instead of relying on certain order in deployments.